### PR TITLE
replace require kludge in runRollup.js

### PR DESCRIPTION
**Pull request by Rich-Harris**
See https://github.com/rollup/rollup/pull/706#issuecomment-243290485

### CODE REVIEW

**Comment by snuggs** [comment]
Awe @Rich-Harris you should~~n't~~ have! :-) <3 👍

File: bin/src/runRollup.js [hunk 1]
```diff
@@ -217,7 +216,7 @@ function bundle ( options ) {
 		handleError({ code: 'MISSING_INPUT_OPTION' });
 	}
 
-	return rollup.rollup( options ).then( bundle => {
+	return rollup( options ).then( bundle => {
```
Comment (snuggs) [inline bin/src/runRollup.js]: So will `rollup.rollup(...)` be a deprecated API? Perhaps the `dist` is named rollup but is merely a module wrapper around `rollup`.

Thanks for all the knowledge. I'm new here but willing to help @Rich-Harris

**Comment by snuggs** [comment]
LGTM 🚢  :shipit:

File: bin/src/runRollup.js (hunk 1 above)
Comment (Rich-Harris) [inline bin/src/runRollup.js]: If you import the namespace `rollup = require('rollup')` or `import * as rollup ...` then you have to do `rollup.rollup`, but if you import it directly (`import { rollup } ...`) then you don't – I guess it's just a matter of personal preference. I'd generally expect people to use `rollup.rollup` in CommonJS land and `rollup` in ES module land (same as `fs.readFile(...)` as opposed to `readFile(...)`) but it's a matter of preference.

The `rollup.rollup` duplication was just me amusing myself, I got a kick out of imagining everyone trying to figure out why the [Magical Mystery Tour](https://www.youtube.com/watch?v=kTOyVcs2jkI) was going round and round in their head while they were coding...

Comment (snuggs) [inline bin/src/runRollup.js]: @Rich-Harris more like the Christmas Hat Cat. I love that guy. SO unfortunate about his neck tho. https://www.youtube.com/watch?v=159ERq6Kf9Y

/cc @GGAlanSmithee

**Pull request merged**